_targets.R

```diff
@@ -26,20 +29,38 @@ list(
   # Identify oldest sites
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
   # insert branching step
-  tar_map(
+  mapped_by_state_targets <- tar_map(
     values = tibble(state_abb = states)%>%
       mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
     tar_target(nwis_inventory, get_state_inventory(sites_info = oldest_active_sites, state_abb)),
     tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
     tar_target(tally, tally_site_obs(nwis_data)),
     tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter)),
-    names = state_abb
+    names = state_abb,
+    unlist = FALSE
   ),
```

Comment:
I think I missed this at first also, but you're missing a `format = "file"` here:

`tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter), format = "file")`

That's the issue that's generating those NA values in the file paths which then causes the map target to fail.